docs/packaging/Risky-capabilities.md

```diff
@@ -0,0 +1,30 @@
+# Risky Capabilities
+
+This page specifically focuses on capabilities that are incompatible with win 32 app isolation. The following capabilities can lower the security offered by win32 app isolation and application developers should refrain from using them when onboarding to win32 app isolation. 
+
+* "uiAccess"
```

Comment:
Why quoting the capability? It's probably better to use \` instead of ". That's how we do it in other docs.